pkgs/top-level/release-lib.nix

```diff
@@ -43,6 +46,11 @@ rec {
   pkgs_x86_64_cygwin = allPackages { system = "x86_64-cygwin"; };
 
 
+  assertTrue = bool:
+    if bool
+    then pkgs.runCommand "evaluated-to-true" {} "touch $out"
+    else pkgs.runCommand "evaluated-to-false" {} "false";
```

Comment:
IMHO such tests are better done at build time (i.e. by having a single job that does `nix-instantiate --eval tests.nix`, probably by adding it to `lib/tests/release.nix`). I don't want to clutter Nixpkgs/NixOS jobsets with lots of tiny Nixpkgs lib tests...